docs/source/flow-project.rst

```diff
@@ -153,20 +211,15 @@ The Project Status
 The :py:class:`~flow.FlowProject` class allows us to generate a **status** view of our project.
 The status view provides information about which conditions are met and what operations are pending execution.
 
-A condition function which is supposed to be shown in the **status** view is called a *label-function*.
+A condition function which will be shown in the **status** view is called a *label-function*.
```

Comment:
Invert sentence order so that it's clear that we're defining a new term.
```suggestion
A *label-function* is a condition function which will be shown in the **status** view.
```